node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodesResponse.java

```diff
@@ -220,6 +222,8 @@ private void toSlime(NodeResources resources, Cursor object) {
         object.setDouble("diskGb", resources.diskGb());
         object.setDouble("bandwidthGbps", resources.bandwidthGbps());
         object.setString("diskSpeed", serializer.toString(resources.diskSpeed()));
+        if (resources.storageType() != NodeResources.StorageType.any)
```

Comment:
Why not always serialize like we do for `diskSpeed`?

Reply:
These responses are becoming bloated, and I don't think this will be used that much. I don't really have a strong opinion on it though.

Reply:
Don't show `diskSpeed` when it is `any` then?

Reply:
Yes, but then I need to check if that works with clients ...

Reply:
None of our clients use anything `requestedResources`

Reply:
Ah, yes it's only used for that. The other problem is that the default for diskSpeed isn't "any" but "fast". Ok, ok. Let me do this properly then.